pod.go

```diff
@@ -85,6 +85,18 @@ var (
 		"The number of requested memory bytes  by a container.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerLimitedCpuCores = prometheus.NewDesc(
+		"kube_pod_container_limited_cpu_cores",
+		"The number of limited cpu cores by a container.",
```

Comment:
Absolutely for it. I would even say to avoid further confusion and to stay as close as possible to the API to do `kube_pod_container_resources_{limits,requests}_{cpu_cores,memory_bytes}`. Let's start that naming right away and I'll fix the existing naming, so we have consistency for the next release.